pdns/dnsdistdist/docs/rules-actions.rst

```diff
@@ -1189,6 +1189,7 @@ The following actions exist.
   * ``aa``: bool - Set the AA bit to this value (true means the bit is set, false means it's cleared). Default is to clear it.
   * ``ad``: bool - Set the AD bit to this value (true means the bit is set, false means it's cleared). Default is to clear it.
   * ``ra``: bool - Set the RA bit to this value (true means the bit is set, false means it's cleared). Default is to copy the value of the RD bit from the incoming query.
+  * ``soaInAuthoritySection``: bool - Place the SOA record in the authority section for a complete NXDOMAIN/NODATA reponse that works as a cachable negative response, rather than the RPZ-style response with a purely informational SOA in the additional section.
```

Comment:
A note about when the change was introduced would be nice, something like:
```
.. versionchanged:: 1.8.0
  Added the ``soaInAuthoritySection`` option.
```
below the function definition.

Reply:
It would also be nice to state that the default is `false`.